packages/filesystem/src/browser/file-dialog/file-dialog-service.ts

```diff
@@ -74,7 +74,7 @@ export class DefaultFileDialogService {
     protected async getRootNode(folderToOpen?: FileStat): Promise<DirNode | undefined> {
         const folder = folderToOpen || await this.fileSystem.getCurrentUserHome();
         if (folder) {
-            const rootUri = new URI(folder.uri).parent;
```

Comment:
@elaihau I wonder why this is the case, we already pass the folder as reference when triggering the file dialog why do we need to get the parent after? Maybe you can clarify or help me understand? Perhaps I am calling the `showSaveDialog` with the incorrect `FileStat`.

Without the change the following was displayed https://github.com/theia-ide/theia/pull/4771#issuecomment-478640031 but after we are able to have the context of what folder we are currently in and have the proper tree.

<img width="678" alt="Screen Shot 2019-04-01 at 7 59 47 PM" src="https://user-images.githubusercontent.com/40359487/55366903-bb9f3680-54b8-11e9-96b1-1b79619abe0d.png">

Reply:
@vince-fugnitto Could you revert this change and try to pass a file URI instead of parent folder? I think it was done on purpose to allow opening siblings folders.